user-guide/modules/ROOT/pages/task-simulation.adoc

```diff
@@ -18,4 +18,107 @@ Creating a real-time simulation of vehicles involves various aspects, including
+----
++ 
+Note:: We do not enter the `geometry`, `units`, `asio`, or `interprocess` library names as these are header-only and do not require any building. To determine which libraries need building, refer to  xref:header-organization-compilation.adoc#_required_compiled_binaries[Required Compiled Binaries].
++
+Warning:: You will get a library name error if you use the `--with` switch with a library that does not require building.
+
+. Wait for the command to finish. This can take a few minutes.
+
+. If you now look in your `Simulation` folder, you will see `include` and `lib` subfolders that contain the files you need to get going on your simulation project.
+
+. Add the appropriate `#include` statements to the start of your new project. Refer to the documentation for the individual libraries for details on all the options available. For example, to use the basic boost:graph[] functionalities:
++
+[source,cpp]
+----
+#include <boost/graph/adjacency_list.hpp>
+----
++
+And if you are using boost:graph[] algorithms:
++
+[source,cpp]
+----
+#include <boost/graph/breadth_first_search.hpp>  // for BFS, as an example
+----
++
+Similarly, to use core boost:units[] features:
++
+[source,cpp]
+----
+#include <boost/units/quantity.hpp>
+#include <boost/units/systems/si.hpp>  // for SI units
+----
++
+And for specific units or systems (for example, the imperial system):
++
+[source,cpp]
+----
+#include <boost/units/systems/imperial/length.hpp>
+----
+
+. Go through a similar decision process for the other libraries: boost:chrono[], boost:geometry[], boost:thread[], boost:asio[], boost:interprocess[], boost:mpi[] and boost:serialization[].
+
+. Remember to set your project's `include` path to point to Boost's header files in your `Simulation\include` folder, and your `library` path to your `Simulation\lib` folder containing the built binaries.
+
+. Write a great real-time simulation!
+
+=== Build using CMake
+
+. If you use CMake rather than B2, you would start by creating a `CMakeLists.txt` file at the root of your Boost source folder:
++
+[source,cpp]
+----
+cmake_minimum_required(VERSION 3.15)  # or the version you have
+project(BoostInstallation)
+
+add_subdirectory(libs/chrono)
+add_subdirectory(libs/graph)
+add_subdirectory(libs/thread)
+add_subdirectory(libs/mpi)
+add_subdirectory(libs/serialization)
+----
++
+This will tell CMake to only process the directories for the specified libraries.
+
+. Navigate to your Boost root folder in your terminal or command prompt and run:
++
+[source,cpp]
+----
+mkdir build && cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=C:\Users\YOURNAME\Documents\Boost\Simulation
+----
++
+This will generate build files in the build folder, and set the installation prefix to the specified folder.
+
+. After the build files are generated, use CMake to build and install:
++
+[source,cpp]
+----
+cmake --build . --target install
```

Comment:
This is the command for installing, not building (although cmake might notice a dependency between two targets).

The cmake script above is also not prepared to install the library. Creating a install target is quite complex and even some good libraries don't provide these. 

A user experimenting with boost will definitely not create a `install` target. They will basically build and run code. Installing it as an application is a complex issue related to cmake only.

Have you run any of these commands and scripts locally? It's hard to believe any step worked at all.